README.md

```diff
@@ -1 +1,46 @@
-# Leaderboard
\ No newline at end of file
+# Leaderboard
+
+> LeaderBoard
+
+## Built With
+
+- HTML
+- JavaScript
+- CSS
+- webpack
+
+## Live Demo
+
+[Live Demo Link]()
+
+
+## Getting Started
+
```

Comment:
- [ ] Please tell users how to set up your project on their local machines. This will make your README even more professional 👍